### PR TITLE
Control up to 8 LEDs

### PR DESCRIPTION
**Pull request by kYwzor**
This tool works perfectly to control 2 of my Omen 45L's LEDs: the Omen logo at the front and the LEDs on the side of the case. However, this model has another 4 LED modules that could be controllable: the CPU pump and three fans at the front.

For my machine, the LED numbering is as follows:

| LED number  | Module      |
| ------------- | ------------- |
| 1                  | Front |
| 2                  | Lateral (aka "chase") |
| 3                  | N/A |
| 4                  | CPU pump block |
| 5                  | Front bottom fan |
| 6                  | Front middle fan |
| 7                  | Front top fan |
| 8                  | N/A |

Because the first two LEDs are numbered the same on both machines, it's possible to create a patch that is fully backwards-compatible.

The changes on this PR enable support for controlling the first 8 LEDs on an Omen machine. I did not attribute any particular naming to these extra LEDs, since it's unclear if the numbering is the same on all machines; instead the LEDs are addressable via their number (see table above).

In principle, supporting even more LEDs should be as simple as changing the N_LEDS variable that I created. I personally tested sending commands of up to 12 LEDs and it did not seem to have any harmful effect (though more than 13 LEDs will definitely not work). All that said, I propose keeping it at 8 to stay on the safe side, at least until a real use-case for >8 shows up.

@chiahsing please let me know if you have any questions/concerns about this PR.

### CODE REVIEW

File: omen_light.cc [hunk 1]
```diff
@@ -47,13 +46,12 @@ struct __attribute__((packed)) Report {
   uint8_t custom_color_count;  // total number of custom colors
   uint8_t custom_color_id;     // the custom color to set. Starting with 1
   uint8_t reserved2[2];
-  Color front_color;
-  Color chase_color;
-  uint8_t reserved3[34];
+  Color colors[N_LEDS];
+  uint8_t reserved3[40 - N_LEDS * 3];
```
Comment (chiahsing) [inline omen_light.cc]: this might just be 40 - sizeof(colors)?

File: omen_light.cc [hunk 2]
```diff
@@ -122,14 +123,27 @@ bool SendReport(const Report& report) {
...
-    *led = Report::Led::FRONT;
+    *led = 1;
   } else if (arg == "chase") {
-    *led = Report::Led::CHASE;
+    *led = 2;
   } else {
-    std::cerr << "unknown led: " << arg << std::endl;
-    return false;
+    try
+    {
```
Comment (chiahsing) [inline omen_light.cc]: I use google style for clang format. Maybe follow the same style here?

**Review by chiahsing — COMMENTED** [review]
Thanks for the patch!

File: omen_light.cc (hunk 1 above)
Comment (kYwzor) [reply omen_light.cc]: Agreed!

File: omen_light.cc (hunk 2 above)
Comment (kYwzor) [reply omen_light.cc]: Good call

**Comment by kYwzor** [comment]
Added a commit fixing the stuff you pointed out. I also added the "unicorn" theme that is available at least in my device. In my experience it also supports up to 6 custom colors, so I changed the code to allow that (currently it was only allowing 4).

**Review by chiahsing — APPROVED** [review]

**Pull request merged**